app/views/publish/courses/withdrawals/edit.html.erb

```diff
@@ -35,14 +35,14 @@
 
       <p class="govuk-body">Once you’ve withdrawn a course, you cannot publish it again this cycle.</p>
 
-      <h2 class="govuk-heading-m">Confirm withdrawal</h2>
-
-      <%= f.govuk_text_field :confirm_course_code, label: { text: "Enter the course code to confirm" }, width: 5 %>
-
-      <%= f.govuk_submit "Yes I’m sure – withdraw this course", class: "govuk-button govuk-button--warning" %>
+      <% if @course.application_status_open? && FeatureService.enabled?(:open_and_closed_course_flow) %>
+        <p class="govuk-body">If you want to stop new candidates from applying but do not want to withdraw any applications currently in progress, you can <%= govuk_link_to("close the course instead", application_status_publish_provider_recruitment_cycle_course_path) %>.
```

Comment:
If you click `close the course instead` then click the back link, you don't go back to the withdraw page.

Shall we have a `goto_withdraw` for the back link?

Reply:
fixed back link optted with `goto` as the key and `withdraw` as value